api/swagger.yml

```diff
@@ -1063,6 +1089,60 @@ paths:
         default:
           $ref: "#/components/responses/ServerError"
 
+  /auth/password:
+      post:
+        tags:
+          - auth
+        operationId: updatePassword
+        summary: Update user password by reset_password token
+        security:
+          - cookie_auth: []
+        requestBody:
+          required: true
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/UpdatePasswordByToken"
+        responses:
+          201:
+            description: successful reset
+          401:
+            $ref: "#/components/responses/Unauthorized"
+          default:
+            $ref: "#/components/responses/ServerError"
+ 
+  /auth/password/forgot:
+      post:
+        tags:
+          - auth
+        operationId: passwordForgot
+        summary: forgot password - request to start password reset flow
+        security:
+          - cookie_auth: []
+        requestBody:
+          required: true
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ForgotPasswordRequest"
+        responses:
+          200:
+            description: password reset request submitted
+          400:
+            description: bad request
+            content:
+              application/json:
+                schema:
+                  $ref: "#/components/schemas/Error"
+          500:
+            $ref: "#/components/responses/ServerError"
+          default:
```

Comment:
keep the server error as default - use 204 instead of 200

Reply:
Ok. fixed